wormhole-connect/src/components/ConnectWallet.tsx

```diff
@@ -121,6 +122,19 @@ function ConnectWallet(props: Props) {
                   >
                     Copy address
                   </div>
+                  {EXPLORER ? (
+                    <div
+                      className={classes.dropdownItem}
+                      onClick={() =>
```

Comment:
I'd suggest moving this to a function like `copy` or `connect`

Reply:
sure thing

Reply:
I've refactored it a little bit, to make it clear and simple